static/locales/en-US.yaml

```diff
@@ -251,6 +251,9 @@ Settings:
     Fallback to Non-Preferred Backend on Failure: Fallback to Non-Preferred Backend
       on Failure
     Enable Search Suggestions: Enable Search Suggestions
+    Auto Load Next Page:
+      Label: Auto Load Next Page
+      Tooltip: This also handles video comment auto load
```

Comment:
```suggestion
      Tooltip: Load comments and additional page results without having to press "Load More" buttons.
```

Reply:
"page results" seems like a strange term...
`Load additional pages and comments without clicks.`? (Can swap `without clicks` with `automatically` not sure which one better

Reply:
I'm good with that (the "automatically" version)

Reply:
Updated